src/sentry/integrations/jira/integration.py

```diff
@@ -822,15 +829,27 @@ def get_create_issue_config(self, group: Group | None, user: RpcUser, **kwargs):
             if not any(c for c in issue_type_choices if c[0] == issue_type):
                 issue_type = issue_type_meta["id"]
 
-        fields = [
+        projects_form_field: dict[str, Any] = {}
+        if features.has("organizations:jira-paginated-projects", group.organization, actor=user):
+            paginated_projects_url = reverse(
+                "sentry-extensions-jira-search", args=[self.organization.slug, self.model.id]
+            )
+            projects_form_field["url"] = paginated_projects_url
+
+        projects_form_field.update(
             {
                 "name": "project",
                 "label": "Jira Project",
-                "choices": [(p["id"], p["key"]) for p in jira_projects],
+                "choices": [(p["id"], f"{p["key"]} - {p["name"]}") for p in jira_projects],
                 "default": meta["id"],
                 "type": "select",
                 "updatesForm": True,
-            },
+                "required": True,
```

Comment:
Was this change to `required: True` intentional?

Reply:
Yeah it was intentional since it felt weird that the project field wasn't a required field to be filled (tbf, I don't think in reality the people can actually leave it blank in the issue create form, but safer imo 🤷‍♀️ )